tests/unit/HelperTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+use Mockery as m;
+use Illuminate\Container\Container;
+
+class HelperTest extends PHPUnit_Framework_TestCase
+{
+	public function setUp()
+	{
+		parent::setUp();
+
+		$store = m::mock(anlutro\LaravelSettings\SettingStore::class);
```

Comment:
We support PHP 5.3 which means this should be a string as the `class` constant does not exist.